src/script/interpreter.cpp

```diff
@@ -220,7 +220,7 @@ bool EvalScript(std::vector<valtype> &stack, const CScript &script,
         return set_error(serror, ScriptError::SCRIPT_SIZE);
     }
     int nOpCount = 0;
-    bool fRequireMinimal = (flags & SCRIPT_VERIFY_MINIMALDATA) != 0;
```

Comment:
This seems wise. Maybe add a TODO or something here for posterity to go and delete the excess tests and branches.

Reply:
How about moving it into some constant somewhere and changing the case to SCREAM_SNAKE?